npm/design-system/cypress/support/component.js

```diff
@@ -2,6 +2,7 @@ import 'regenerator-runtime/runtime'
 import 'cypress-real-events/support'
 import '@percy/cypress'
 import './storybook'
+import 'normalize.css/normalize.css'
```

Comment:
I don't think we need this do we? We are not consuming the design-system package. Should this be in frontend-shared?

Also, do we need it at all? IIRC tailwind does a css reset anyway.

Reply:
We import normalize.css in frontend-shared. I added this here because I removed it from `global.scss` in the design system and I didn't want there to be any changes when it comes to testing the design system components since they still run on CI.

We are indirectly consuming the design system. We import the runner, and the runner uses components from the design system which pulls in a second copy of normalize (even though we don't use the components, normalize was still being bundled and overriding our overrides). Removing it from `global.scss` fixed this issue